connectivity-proxy/README.md

```diff
@@ -1,148 +1,156 @@
-## SAP Connectivity Proxy
+# Connectivity: Accessing a Workload in the Corporate Network From SAP BTP, Kyma Runtime
 
-The sample demostrates the use of the SAP Cloud Connector from within SAP BTP, Kyma runtime. This includes...
+## Context
 
-- The provisioning of the Connectivity Proxy within the Kyma runtime ([SAP Help Portal link](https://help.sap.com/docs/BTP/65de2977205c403bbc107264b8eccf4b/0c035010a9d64cc8a02d872829c7fa75.html?locale=en-US)).
-- A sample Nodejs application is ran locally and configured in the SAP Cloud Connector to be exposed to the connected SAP BTP account.
-- A Serverless function is deployed within the Kyma runtime, which is configured to call the sample Nodejs application via the Connectivity Proxy.
-- Using a curl pod to call the sample Nodejs application via the Connectivity Proxy.
+If you want to access a workload in the corporate network from SAP BTP, Kyma runtime, you can use Connectivity Proxy from SAP BTP, Connectivity. SAP BTP, Connectivity also provides Cloud Connector service channels to connect your on-premises network to specific services on SAP BTP, Kyma runtime.
```

Comment:
We can either leave this part out as this sample is for cloud to on-premise. servcie channels are about on-premise to cloud.

or clearly state why service channels are used and link to that other sample